app/js/data/iconListInfo.js

```diff
@@ -0,0 +1,62 @@
+// JSON Object of all of the icons and their tags
```

Comment:
For future consideration: you can imagine in the future that this list might be loaded asynchronously from a database. You might consider having this module export a node-style async function:

```js
export default function (err, cb) {
  let iconList = { 
    // ...
  };
  cb(iconList);
}
```